app/assets/stylesheets/application.css.scss

```diff
@@ -533,4 +533,59 @@ span.twitter-typeahead {
 
 .latest-run--ok {
   border-color: #5cb85c;
+}
+
+.alert-wrap {
+  text-align: center;
+  margin-bottom: 1.5em;
+
+  .alert {
+    display: inline-block;
+    margin: 0 auto;
+  }
+}
+
+.alert-icon {
+  &.alert-success,
+  &.alert-info,
+  &.alert-warning,
+  &.alert-danger {
+    background-color: transparent;
+  }
+
+  position: relative;
+  padding-top: 10em;
+
+  &:before {
+    content: '';
```

Comment:
Properties should be ordered background-position, background-repeat, background-size, border, border-radius, content, display, height, margin-left, position, top, width<br>Prefer double-quoted strings